README.md

```diff
@@ -1,87 +1,105 @@
-# Transparent Release of Binaries
+# The Transparent Release Project
+
+Transparent Release (not really the best name for the project) builds on the idea of binary
```

Comment:
"(not really the best name for the project)" --- I'd drop that. It is a bit confusing/discouraging as a first point. If you want to keep it: Can we write the history instead on why the name developed?